addons/account_peppol/wizard/account_move_send_batch_wizard.py

```diff
@@ -0,0 +1,12 @@
+from odoo import models
+
+
+class AccountMoveSendWizard(models.TransientModel):
+    _inherit = 'account.move.send.batch.wizard'
+
+    def action_send_and_print(self, allow_fallback_pdf=False):
+        # EXTENDS 'account'
+        self.ensure_one()
+        if self.sending_method == 'peppol':
+            self._do_pre_send()
```

Comment:
Indeed, I was thinking it could be something that become generic in `account`, but maybe it's overreacting since it's only usefull in Peppol for now...